### ISSUE TITLE
Formatting removes all indentation

### ISSUE THREAD

**Issue by donaldhubduck** [onecentlin/laravel-blade-snippets-vscode#104]
I have event tried this on a clean install of Ubuntu on a virtual machine with only vs code with this extension. As soon as I format the document it removes all indentation.

Before formatting:
`@extends('layouts.app')

@section('content')
  @while(have_posts()) @php the_post() @endphp
    @include('partials.page-header')
    @include('partials.content-page')
  @endwhile
@endsection`

After formatting
`@extends('layouts.app')

@section('content')
@while(have_posts()) @php the_post() @endphp
@include('partials.page-header')
@include('partials.content-page')
@endwhile
@endsection`

Thanks! :)

**Comment by kylemilloy** [comment]
Yeah this is something I've been ignoring but as I get more and more nested blade directives in a page I'd want to see them nested. A good example where this makes it completely unscannable is for, break, continue loops...

```
@section('content')
  @for(range(0,10) as $n)
    @if($n === 0)
      <div>
        Header
      </div>
      @continue
    @endif

    @unless($n > 9)
      <div>
        {{ $n }}
      </div>
      @break
    @endif
  @endfor
@endsection
```

Gets turns into

```
@section('content')
@for(range(0,10) as $n)
@if($n === 0)
<div>
  Header
</div>
@continue
@endif

@unless($n > 9)
<div>
  {{ $n }}
</div>
@break
@endif
@endfor
@endsection
```

**Comment by donaldhubduck** [comment]
Is it better that I close this issue and continue on #92 ? So that the issue gets more attention? Can you kylemilloy also post what you posted here, on that issue #92 too?

**Comment by josuecatalan** [comment]
Same for me, exist othe blade formatter to vs code? i can't find one :(

**Comment by Slivicon** [comment]
also discussed before in #61

**Comment by alexkerber** [comment]
Any more updates with this? I really just want to use this plugin and not involve beautify.

**Comment by iRaziul** [comment]
Its too annoying -_-

**Comment by diontron** [comment]
still unusable because of this issue in 2022

**Comment by kylemilloy** [comment]
If you're just looking for the formatter, this is what I've been using.
https://github.com/shufo/blade-formatter

It has it's own [issues](https://github.com/shufo/vscode-blade-formatter/issues/254) but it works far better.

**Comment by martin-braun** [comment]
https://github.com/shufo/vscode-blade-formatter/

This is the best one on the market, so far. Unfortunately, there is no blade formatter without issues, but the author of the one I mention here is very active
